src/dict.c

```diff
@@ -295,7 +298,7 @@ int dictTryExpand(dict *d, unsigned long size) {
 int dictRehash(dict *d, int n) {
     int empty_visits = n*10; /* Max number of empty buckets to visit. */
     if (dict_can_resize == DICT_RESIZE_FORBID || !dictIsRehashing(d)) return 0;
-    if (dict_can_resize == DICT_RESIZE_AVOID && 
```

Comment:
If you can @vitarb, it would probably be a good time to go through and remove all of these unnecessary touch points.

Reply:
Can you please clarify what you mean exactly?

Reply:
Whitespace changes on otherwise untouched lines.

Reply:
Of course, my IDE seem to be overly eager at removing unnecessary spaces at the EOL.